charts/stable/cloudreve/docs/installation_notes.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Important Notes
+---
+
+## Credentials
+
+On first run, cloudreve will generate an admin user and print the email and password to the app logs. To retrieve, wait for the app to finish deploying, and then click the `View Logs` icon in the `Workload` card.
+
```

Comment:
Lets rephrase this, something like:
Check the logs for the email and password, in TrueNAS SCALE you can find it by clicking xxxx

We have to take into account that the chart might be deployed in a non scale platform

Reply:
Thanks. Took a pass at this change in https://github.com/truecharts/charts/pull/17489/commits/303a1761d74a54821d37a36bcd621581f1aa5538.